crates/vm/src/sync.rs

```diff
@@ -157,6 +162,61 @@ pub fn step_op(
     }
 }
 
+/// Step forward execution by the given synchronous operation.
+/// This includes the synchronous state read operation.
+pub fn step_any_op<S>(
```

Comment:
I guess we'll change this to `step_op` and remove the old one, `OpSync` and `OpAsync` when we switch to sync-only execution :ok_hand: